test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -1269,6 +1271,25 @@ public Task InheritsOnViewImportsWorksForPagesWithModel()
             Assert.Equal(expected, content);
         }
 
+        [Theory]
+        [InlineData("FromHandlerMethod")]
+        [InlineData(nameof(IAuthorizationFilter.OnAuthorization))]
```

Comment:
Let's not add combinatorial functional tests. Adding unit tests for this is not difficult - https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Internal/PageActionInvokerTest.cs

I'd add one auth or resource filter and one page filter functional test. Also this is not a good TheoryData candidate. Each of them is a separate scenario, so separate tests.